### DISCUSSION TITLE
A calculator application

### DISCUSSION THREAD

**Discussion by YesTheProxy** [theshmunguss1/RamOS#4] [Ideas]
https://dunizb.github.io/sCalc/ is modled after macOS

**Comment by YesTheProxy** [comment]
plus it is in (html, css, javascript)

**Comment by Sebas17Dev** [comment]
Hello, I hope you are well, I modified the CSS file of the calculator project a little, I added some effects to give more feedback to the user, in addition to using CSS variables so that it was more organized. I hope it is to your liking 💙

[New styles.zip](https://github.com/ramnetos/RamOS/files/15221725/New.styles.zip)

**Reply by DJshelfmushroom** [reply]
make a pull request to have your changes considered

**Reply by YesTheProxy** [reply]
ty man

**Reply by DJshelfmushroom** [reply]
NO HE HAS TO MAKE A PULL

**Reply by YesTheProxy** [reply]
yeah, I'm just saying ty for making it
